docs/jaxpr.rst

```diff
@@ -34,17 +34,17 @@ There are two related representations in the code for jaxprs,
 :py:class:`jax.core.Jaxpr`, and is what you obtain when you use
 :py:func:`jax.make_jaxpr` to inspect jaxprs. It has the following fields:
 
-  * ``jaxpr``: is a :py:class:`jax.core.Jaxpr` representing the actual
+  * ``jaxpr`` is a :py:class:`jax.core.Jaxpr` representing the actual
     computation content of the function (described below).
   * ``consts`` is a list of constants.
 
 The most interesting part of the ClosedJaxpr is the actual execution content,
 represented as a :py:class:`jax.core.Jaxpr` as printed using the following
 grammar::
 
-   jaxpr ::= { lambda Var* ; Var+.
-               let Eqn*
-               in  [Expr+] }
+ Jaxpr ::= { lambda Var* ; Var+.
+             let Eqn*
+             in  [Expr+] }
```

Comment:
Should that be

```suggestion
 Jaxpr ::= { lambda Var* ; Var+. let
               Eqn*
             in  [Expr+] }
```

?

Reply:
Also, the indentation should be 2 chars. It's now 1 char here.

Reply:
Thanks – done.